x-pack/plugins/cloud_security_posture/server/create_transforms/latest_vulnerabilities_transforms.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
+import {
+  CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
+  LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
+  VULNERABILITIES_INDEX_PATTERN,
+} from '../../common/constants';
+
+export const latestVulnerabilitiesTransform: TransformPutTransformRequest = {
+  transform_id: 'cloud_security_posture.vulnerabilities_latest-default-8.8.0',
+  description:
+    'Defines vulnerabilities transformation to view only the latest vulnerability per resource',
+  source: {
+    index: VULNERABILITIES_INDEX_PATTERN,
+  },
+  dest: {
+    index: LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
+  },
+  frequency: '5m',
+  sync: {
+    time: {
+      field: 'event.ingested',
+      delay: '60s',
+    },
+  },
+  retention_policy: {
+    time: {
+      field: '@timestamp',
+      max_age: '5h',
```

Comment:
@tinnytintin10 / @nick-alayil this retention_policy flag is the time to live the transformed data, 

We initially copied the value of 5h from the cspm/kspm transforms, but I saw we are going to increase it [here](https://github.com/elastic/kibana/pull/152226) to 26hr. Do we have an estimation for each vulnerability management scanning cycle?

Reply:
I agree, vulnerabilities doesn't have the same use case for the retention policy.
I would add more than 26 hours. At least 48h in the first place.

@nick-alayil / @tinnytintin10 how about changing that to `7d` (7-days). Wouldn't that make more sense? to automatically remove after 7 days we haven't got any updates? the user is also able to sort by it. And I think it makes the UX more comprehensive